### PR TITLE
Seed scaling

### PR DESCRIPTION
**Pull request by Chris-Hesterman**
App still works, Client tests all passing .

data generation for NoSQL and SQL had to be done separately, so it was. those are done, data generated, and mongo, the inherited db has been seeded.
Screen Shot 2020-08-29 at 18 55 54 PM

### CODE REVIEW

**Comment by Chris-Hesterman** [comment]
I meant add back code necessary for testing ^

**Comment by Chris-Hesterman** [comment]
Seed.js is now obsolete, as seeding is done with the scripts - seedMongo1 and seedMongo2.

File: server/seed/seed.js [hunk 1]
```diff
@@ -705,7 +705,7 @@ const generateRecord = function () {
 //Initiate randomly generated data - to increase/decrease amount of randomly generated records, alter the number itemIdTracker is being compared to
 /////////////////////////////////////
 if (process.env.node_env !== 'test') {
-  while (itemIdTracker <= 199) {
+  while (itemIdTracker <= 50099) {
```
Comment (WDHudson) [inline server/seed/seed.js:708]: Why 50099?

**Review by WDHudson — APPROVED** [review]
It looks great! You're killing it!

File: server/seed/seed.js (hunk 1 above)
Comment (Chris-Hesterman) [reply server/seed/seed.js:708]: That is inherited seeder/generator that is now obsolete.  As for the answer?  got me.

**Pull request merged**